<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>product</class>
 <widget class="QDialog" name="product">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>761</width>
    <height>329</height>
   </rect>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::DefaultContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>611</width>
     <height>271</height>
    </rect>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::ActionsContextMenu</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Edwardian Script ITC</family>
     <pointsize>28</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Administrador</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>50</y>
     <width>120</width>
     <height>141</height>
    </rect>
   </property>
   <property name="title">
    <string>Categoria PRODUCTO</string>
   </property>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>101</width>
      <height>111</height>
     </rect>
    </property>
    <property name="text">
     <string>1. Bebidas
2. Carnes
3. Verduras
4. Lacteos
5. Granos
6. Frutas
7. Aseo
8. Otros</string>
    </property>
   </widget>
  </widget>
  <action name="actionAdd">
   <property name="text">
    <string>Agregar Producto</string>
   </property>
  </action>
  <action name="actionDelete">
   <property name="text">
    <string>Eliminar Producto</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
